railties/lib/rails/source_annotation_extractor.rb

```diff
@@ -58,7 +58,7 @@ def to_s(options = {})
       # Used in annotations.rake
       #:nodoc:
       def self.notes_task_deprecation_warning
-        ActiveSupport::Deprecation.warn("This rake task is deprecated and will be removed in Rails 6.1. \nRefer to `rails notes --help` for more information.\n")
+        ActiveSupport::Deprecation.warn("This Rails command is deprecated and will be removed in Rails 6.1. \nRefer to `rails notes --help` for more information.\n")
```

Comment:
Here should be rake task.

Reply:
So true... 😮 Good catch @rafaelfranca 

Changing it.